service-a/src/main/java/com/example/services/CustomerServiceFeignClient.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.example.services;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+
+/**
+ *
+ * @author roland.werner
+ */
+@FeignClient(value = "serviceb")
```

Comment:
Übrigens muss es "serviceB" heißen, nicht "serviceA", da man hier hinschreibt, welchen Service man aufruft (in Service A sind wir ja selbst).